documentation/manual/compile.rst

```diff
@@ -19,3 +19,13 @@ you can find examples in the ``config`` folder.
 
    compile-with-cmake
    compile-with-make
+   
+
+A Docker image is also available to try out our code. After installing Docker
+(https://docs.docker.com/install/) you can run the following command: 
+
+   docker run -it -v "$PWD":/home/mightybuilder/work handeqmc/hande_ubuntu18.04:latest
```

Comment:
Didn't we discuss removing the mention of the `-v "$PWD":/home/mightybuilder/work` part of the command?

Reply:
Have split it into two